include/ola/client/ClientTypes.h

```diff
@@ -29,6 +29,8 @@
 #include <string>
 #include <vector>
 
+#include "common/protocol/Ola.pb.h"
```

Comment:
I am doing this at the moment. The only problem is that

    static OlaInputPort FromProtobuf(ola::proto::PortInfo port_info);

and

    static OlaInputPort FromProtobuf(ola::proto::PortInfo port_info);

can't coexist.